overrides/overrides.go

```diff
@@ -0,0 +1,66 @@
+// Package overrides handles managing overrides in the running application
+package overrides
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Masterminds/glide/msg"
+	gpath "github.com/Masterminds/glide/path"
+)
+
+var overrides map[string]*override
```

Comment:
That is a great question. I'm pretty sure there is so I'm not sure why it's not complained. I'll fix this. Thanks.